docs/thinking-in-atomic.md

```diff
@@ -14,7 +14,7 @@ title: Thinking in Atomic
 Adopting an Atomic CSS methodology addresses common CSS challenges:
 
 <dl class="Mb(30px)">
-    <dt class="Fs(i) C(#000)">Changes are predictable</dt>
+    <dt class="Fs(i) C(--color-emphasis)">Changes are predictable</dt>
     <dd class="Mstart(20px) Mt(5px)">Because of the single responsibility principle (one class == one style) it is easy to predict what removing or adding a class will do.</dd>
     <dt class="Fs(i) C(#000)">Scope is limited</dt>
```

Comment:
`--color-emphasis` ?

Reply:
Good find! We have custom css to style the `<dt>` tag so the `C()` class isn't actually needed. Removed in https://github.com/acss-io/atomizer/pull/505/commits/68ae81e63bbde3fcc9dc07626fc153884b6c7a97